tests/cli/test_blueprints_sanity.sh

```diff
@@ -41,5 +43,30 @@ __EOF__
         rlAssertEquals "pushed bp is found via list" "`$CLI blueprints list | grep beakerlib`" "beakerlib"
     rlPhaseEnd
 
+    rlPhaseStartTest "blueprints show"
+        rlAssertEquals "show displays blueprint in TOML" "`$CLI blueprints show beakerlib`" "`cat beakerlib.toml`"
+    rlPhaseEnd
+
+    rlPhaseStartTest "SemVer .patch version is incremented automatically"
+        # version is still 0.0.1
+        rlAssertEquals "version is 0.0.1" "`$CLI blueprints show beakerlib | grep 0.0.1`" 'version = "0.0.1"'
+        # add a new package to the existing blueprint
+        cat >> beakerlib.toml << __EOF__
+
+[[modules]]
```

Comment:
As I mentioned in the other one, this should really be [[packages]] since [[modules]] is going to change to actual modules at some point. If it is failing we need to figure out why.